components/ProgressBar.tsx

```diff
@@ -0,0 +1,16 @@
+type ProgressBarProps = {
+    progress: number;
+    showLabel: boolean;
+    appendLabelText: string;
+};
+
+const ProgressBar = ({ progress, showLabel, appendLabelText }: ProgressBarProps) => {
+    return (
+        <div className="progress-container">
```

Comment:
Can we update to the HTML standard `<progress>`?

Reply:
You might be right. I thought custom styling on <progress> would be tricky in a cross-browser compatible way, especially since I animated with a CSS transition the jumps in percentages after each dot is completed, but it looks like it's mostly possible now. Let me double check and will update.